core/runtime.rs

```diff
@@ -1385,7 +1386,14 @@ impl JsRuntime {
       return Ok(());
     }
 
-    let key = { *state.pending_promise_exceptions.keys().next().unwrap() };
+    let key = {
+      state
+        .pending_promise_exceptions
+        .keys()
+        .next()
+        .unwrap()
+        .clone()
+    };
```

Comment:
Just curious, is there a reason fro the braces? Is it stylistic?

Reply:
Borrow checker complains about double borrow of `state` if there are no braces.